src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -172,18 +170,10 @@
       updateDefender(null);
       updateAttacker(null);
     }
-    if (OddsCalculatorPanel.percentageOfFreeMemoryAvailable() < 0.4) {
-      System.gc();
-      System.runFinalization();
-      System.gc();
-    }
```

Comment:
This kind of heuristics is scary 😱 
I'd argue that in java the GC should be able to figure this out on its own.

Reply:
One thing to consider though: It might make sense to invoke the garbage collector after an oddscalculator run from the Panel, to clean everything up afterwards

Reply:
It's correct to remove, there is no guarantee System.gc would actually gc. The 'manual' gc is a hack, likely one that does not work. It's much better to ensure objects have short scopes, short lived objects are very cheap.